index.html

```diff
@@ -671,10 +671,241 @@
 			</section>
 
-			<section id="enh-pagination">
-				<h3>Pagination</h3>
+			<section id="Layout">
+				<h3>Layout</h3>
+
+				<p class="issue advisement">This section needs thorough review not only from the publishing community
+					to ensure that the proposed behavior is desirable,
+					but also from Web Browser vendors and other implementors of layout engines
+					to ensure that is it implementable.</p>
+
+				<p>The layout and rendering of <a>Web Publications</a> is governed by the usual rules that apply to all web content.
+				[[!HTML]] (including XHTML) documents are styled and laid out according to the rules of <a href="https://www.w3.org/TR/CSS/">CSS</a>,
+				[[!SVG]] files are rendered as normally expected of that format, and so on.
+				This specification requires no particular profile or subset of CSS, HTML, or SVG to be supported,
+				other than the expectations set for these technologies by their respective specifications.</p>
+
+				<div class=note>
+					<p>This specification intentionally refrains from introducing any new layout feature.
+					Any shortcoming of the web platform in terms of layout needs to be addressed for the whole web platform,
+					which means via CSS.</p>
+
+					<p>This working group recognizes that the features offered by CSS specifications have limitations,
+					and that in some cases implementations may be incomplete.
+					However, attempts to address these shortcomings for the exclusive purpose of <a>Web Publications</a>
+					would at best have no effect,
+					and at worst lead to a divergence between “ordinary” web content and <a>Web Publications</a>,
+					instead of the convergence this specification aims to achieve.</p>
+
+					<p>This working group will work with other relevant groups of the W3C to address
+					platform wide limitations that negatively impacts <a>Web Publications</a>,
+					and invites readers to do the same.</p>
+				</div>
+
+				<p>For the purposes of layout, each resource of a <a>Web Publication</a> is treated as a separate document.
+				User Agents must not mix content from multiple resources in the same rendering:
+				For instance, <a href="https://www.w3.org/TR/CSS21/visuren.html#floats">CSS floats</a> or <a href="https://www.w3.org/TR/CSS21/visuren.html#absolutely-positioned">Absolutely positioned elements</a>
+				from one resource may not intrude or overlap with content from an other resource.</p>
+
+				<p>Users must therefore rely on the UI features of the User Agent
+				to navigate from one resource to another,
+				rather than expect a collection of resources to be readable as if it were a single one.</p>
+
+				<p>This does not imply that User Agents are forbidden from juxtaposing the rendering of the various resources.
+				Scrolling User Agents may for instance place renderings of the resources beneath each other in the <a>default reading order</a>,
+				so that users may seamlessly scroll past the end of one and into the next one,
```

Comment:
Why beneath?  That seems very stringent on implementation

Reply:
"may for instance" this is an example, not a constraint. UA innovations is more than welcome here.